src/App.tsx

```diff
@@ -23,30 +23,10 @@ import { Header } from './components/Header.tsx';
 import { PublicRiddlesPage } from './pages/PublicRiddlesPage.tsx';
 import { ReceivedRiddlesPage } from './pages/ReceivedRiddlesPage.tsx';
 import { MyRiddlesPage } from './pages/MyRiddlesPage.tsx';
-
-export const theme = createTheme({
-	palette: {
-		mode: 'dark',
-		background: {
-			default: '#001233'
-		},
-		primary: {
-			main: deepOrange[500] //TODO: replace
-		},
-		secondary: {
-			main: red[400]
-		},
-		text: {
-			//
-		}
-	},
-	typography: {
-		fontFamily: ['"Baloo 2"', 'Roboto'].join(', ')
-	}
-});
+import { theme } from './theme.ts';
 
 export const App = () => {
-	const x = 'AOJ ÉO!';
+	const x = 'RIDDLESHARE';
 
 	return (
```

Comment:
`CssBaseline` jsem znal pouze jako self-closing tag a tedy, že se CSS baseline používá globálně pro celou appku. Díky tvému řešení jsem se ale dozvěděl, že se dá použít i tak, jak jsi to udělal. Nicméně, pokud jsem to pochopil správně, tak se tento způsob používá spíše pro nějaké scoped CSS (např. odlišná část aplikace s jinými styly). Myslím, že to ale nepotřebujeme, a tak bych byl pro použití self-closing tagu na jeden řádek.
```
<CssBaseline />
```
Opět to není chyba, ale o jeden level menší zanoření opět o něco zvýší čitelnost.

Viz https://mui.com/material-ui/react-css-baseline/#scoping-on-children a sekce níže 'Color scheme'

Reply:
OK